### DISCUSSION TITLE
connection/list Ooops, something bad has happened!

### DISCUSSION THREAD

**Discussion by ghost** [apache/airflow#21258] [Q&A]
### Apache Airflow version

2.2.1

### What happened

The UI doesn't display my connections list. I run the scrypt below, and copied the fernet key in my airflow.cfg
    #from cryptography.fernet import Fernet
    #fernet_key = Fernet.generate_key()
    #print(fernet_key.decode())
Even after pasting the generated key I get: 
File "/usr/local/lib/python3.8/dist-packages/airflow/models/connection.py", line 238, in get_password
    return fernet.decrypt(bytes(self._password, 'utf-8')).decode()
  File "/usr/local/lib/python3.8/dist-packages/cryptography/fernet.py", line 194, in decrypt
    raise InvalidToken
cryptography.fernet.InvalidToken

I am also having connection refused: "AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")" and "ERROR - Unable to retrieve connection from secrets backend (MetastoreBackend)."
 

### What you expected to happen

It was working just fine, and overnight airflow stopped working and throwing this error. I redefined the connections, yet still.

### How to reproduce

I did it as follows: I created a new dag that only prints "hello" to a postgres database and used a connection I had already (one that worked just fine the day before), and the error comes up.

### Operating System

Python version: 3.8.9

### Versions of Apache Airflow Providers

Airflow version: 2.2.1

### Deployment

Official Apache Airflow Helm Chart

### Anything else

Node: staging-sql

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by sammed-ai** [comment]
Same issue error on both the cases. Did anyone find any solution to this

**Comment by ghost** [comment]
I keep having the same issue... -- I opened a question in stackoverflow https://stackoverflow.com/questions/70770788/airflow-oops-something-bad-has-happened

**Comment by grzes71** [comment]
See: [https://github.com/aws/aws-mwaa-local-runner/issues/24](https://github.com/aws/aws-mwaa-local-runner/issues/24)
`airflow db reset` worked for me

**Reply by morningcloud** [reply]
airflow db reset will clear all existing dag run history, isn't it?

**Reply by alexisaraya** [reply]
it helped me too

**Comment by naveenbussari** [comment]
airflow db reset will clear the existing dag runs' data. We cannot lose that.
Is there any other solution??
